server/server.py

```diff
@@ -166,8 +173,10 @@ def wrong_argument(message):
 def get_version():
   # Verify that tensorboard is running
   req_res = tb_request("logdir")
-  if not isinstance(req_res, str):
-    return req_res
+  try:
```

Comment:
This one is not a `request.args.get`. There is no string issue here.
The current issue is that right now, since `tb_request` can return an error [here](https://github.com/torrvision/crayon/blob/a1573d5c7d84e0e429f4efac00ac1ba61e425bb6/server/server.py#L114-L115), if its result is going to be used, we need to first check that it did not failed as is done [here](https://github.com/torrvision/crayon/blob/a1573d5c7d84e0e429f4efac00ac1ba61e425bb6/server/server.py#L184-L186) as well.
This behavior could be changed by making `tb_request` raise an error that we could catch here. That would remove a function that can return different types (that is not good :D). But that would mean protecting all the `tb_request` calls in try/except.